README.md

```diff
@@ -15,13 +41,378 @@ The cctbx is designed with an open and flexible architecture to promote extendab
 
 Use of the Python interfaces is highly recommended, but optional. The cctbx can also be used purely as a C++ class library.
 
-## Installation
 
-The easiest way to set up a development environment from scratch is to:
+<a name="install"></a>
+## Installation (Mac OS / Linux)
+
+Current efforts are to incorporate cctbx into a conda environment. Here is a basic installation workflow. Note, cctbx is closely linked to the DIALS diffraction software package, and the DXTBX software package, which provide algorithms for processing experimental diffraction data. Therefore this build will include builds of DIALS and DXTBX, as well as a few others. 
+
+
+<a name="getboot"></a>
+### Download the installer script
+
+The installer script [bootstrap](https://raw.githubusercontent.com/cctbx/cctbx_project/darwin_omp/libtbx/auto_build/bootstrap) downloads dependencies and builds the software. Place it in your main working directory, e.g. ```~/crystal```
+
+```
+mkdir ~/crystal  # make a working directory
+cd ~/crystal
+wget https://raw.githubusercontent.com/cctbx/cctbx_project/darwin_omp/libtbx/auto_build/bootstrap
+``` 
+
+Make ```bootstrap.py``` an executable and look at the options (we'll just use a few)
+
+```
+chmod +x bootstrap.py
+./bootstrap.py --help
+```
+
+Note, in the past bootstrap required that the first python binary in your path be python version 2, or else it will fail. Verify this by running
```

Comment:
Does `bootstrap.py` not run under Python 3? 🤔 @bkpoon I am pretty sure this was fixed up in the CCTBX Python 3 migration work?